### DISCUSSION TITLE
Discussion and Questions

### DISCUSSION THREAD

**Discussion by maxspahn** [maxspahn/gym_envs_urdf#224] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by behradkhadem** [comment]
Wow that fast! 😅 

My name is Behrad, I'm born and raised in Iran. I was a mechanical engineering student and I'm currently studying applied design in Sharif university of technology. During covid I fell in love with programming and kinda fell out of love with mechanical engineering. So, the best thing I could do was to continue my academic journey in robotics (to feel that my 5 years of life wasn't a waste of time 😅 ). I currently write mobile application via flutter for an Australian company.
I love robotics and I'm here because my masters project is reinforcement learning in robotics. 

I hope I help developing this wonderful package which is both unique and useful.
